### PR TITLE
PDQm: replace the ITI-78 transaction with the ITI-119 transaction

### LINKED ISSUE #224: Patient Demographics Query for Mobile (PDQm) (OIiver Egger, ahdis ag)
ch.fhir.ig.ch-epr-fhir#4.0.1-ballot /iti-pdqm.html

Currently the Swiss Specification requires ITI-78 Mobile Patient Demographics Query for the UseCase of Querying patient demographics. This works will, has however the disadvantage that the returned Patient will contain all identifiers assigned to the patient (local identifiers for the patient). 

IHE ITI introduced the  Patient Demographics Match [ITI-119] transaction, which would defines an operation where the result could be enforced to contain only the EPRS-PID identifier.

Propose that the transaction been switched.

OIiver Egger, ahdis ag

### PR DESCRIPTION
**Pull request by qligier**
Fixes #224

For review.

Build: https://build.fhir.org/ig/ehealthsuisse/ch-epr-fhir/branches/224-patient-demographics-query-for-mobile-pdqm-oiiver-egger-ahdis-ag/
QA: https://build.fhir.org/ig/ehealthsuisse/ch-epr-fhir/branches/224-patient-demographics-query-for-mobile-pdqm-oiiver-egger-ahdis-ag/qa.html

### CODE REVIEW

File: input/fsh/pdqm.fsh [hunk 1]
```diff
@@ -39,6 +39,118 @@ Description: "The patient demographics and identifier information which can be p
...
+
+// https://github.com/IHE/ITI.PDQm/blob/main/input/fsh/PDQmMatch.fsh
+Profile: CHPDQmMatchInput
+Parent: https://profiles.ihe.net/ITI/PDQm/StructureDefinition/IHE.PDQm.MatchInputPatient
+Title: "CH PDQm Patient Profile for $match Input"
+Description: """
+The PDQm Patient Profile for $match Input SHALL be provided as input to the ITI-119 transaction.
+- While it is not REQUIRED that the input to $match be a valid FHIR instance, it is RECOMMENDED to supply as many elements as possible to facilitate matching.
+- modifierExtension and implicitRules SHALL not be specified.
+- The patient-mothersMaidenName extension is available to hold the mother's maiden name
```
Comment (oliveregger) [inline input/fsh/pdqm.fsh:144]: here we should add the birthName extension instead of patient-mothersMaidenName

File: input/fsh/pdqm.fsh [hunk 2]
```diff
@@ -39,6 +39,118 @@ Description: "The patient demographics and identifier information which can be p
...
+"""
+* name MS                                     // LivingSubjectName
+* gender MS                                   // LivingSubjectAdministrativeGender
+* birthDate MS                                // LivingSubjectBirthTime
+* address MS                                  // PatientAddress
+* identifier MS                               // LivingSubjectId
+* extension[MothersMaidenName] MS             // MothersMaidenName
```
Comment (oliveregger) [inline input/fsh/pdqm.fsh:151]: we should also add extension birthName instead of MothersMaidenName

File: input/pagecontent/iti-119.md [hunk 3]
```diff
@@ -19,24 +19,28 @@ a patient participating in the Swiss EPR.
...
 
 The _Patient Demographics Supplier_ shall follow the expected actions as defined in
-[2:3.78.4.1.3](https://profiles.ihe.net/ITI/PDQm/ITI-78.html#2378413-expected-actions), with the following restrictions:
+[2:3.119.4.1.3](https://profiles.ihe.net/ITI/PDQm/ITI-119.html#23119413-expected-actions), with the following restrictions:
 
-- If there are more than 5 matches, the result should return zero matches with an OperationOutcome requesting more
+- If there are more than 5 matches, the result should return zero matches with an OperationOutcome requesting more 
   query parameters.
+- If the client requests more than 5 matches, the result should return zero matches with an OperationOutcome 
+  explaining the server can't fulfill the request.
```
Comment (oliveregger) [inline input/pagecontent/iti-119.md:44]: - Only Patient identifiers from the community (MPI-PID and EPR-SPID) SHALL be returned.

**Comment by oliveregger** [comment]
tbd: we should add a similar comment to the PIXm response about the identifieres (only allowing EPR_SPID and MPI-PID).

**Review by oliveregger — CHANGES_REQUESTED** [review]
see comments

File: input/fsh/pdqm.fsh (hunk 2 above)
Comment (qligier) [reply input/fsh/pdqm.fsh]: Fixed. I sliced `name`, as it's done in _CHPDQmPatient_

File: input/pagecontent/iti-119.md (hunk 3 above)
Comment (qligier) [reply input/pagecontent/iti-119.md:45]: Added.

File: input/fsh/pdqm.fsh (hunk 1 above)
Comment (qligier) [reply input/fsh/pdqm.fsh]: Fixed,

**Comment by qligier** [comment]
Updated as commented.

**Review by oliveregger — APPROVED** [review]

**Comment by oliveregger** [comment]
> tbd: we should add a similar comment to the PIXm response about the identifieres (only allowing EPR_SPID and MPI-PID).

not necessary as only the EPR-SPID and MPI-PID are returned

**Pull request merged**